package.json

```diff
@@ -26,16 +26,22 @@
   },
   "homepage": "https://github.com/kaiyuanshe/OSS-bot#readme",
   "dependencies": {
+    "@chatie/tsconfig": "^0.6.1",
     "@hapi/hapi": "^18.3.1",
+    "@types/hapi__hapi": "^18.2.5",
     "@types/raven": "^2.5.1",
     "@types/read-pkg-up": "^6.0.0",
-    "@chatie/tsconfig": "^0.6.1",
-    "@types/hapi__hapi": "^18.2.5",
     "brolog": "^1.6.5",
+    "commander": "^4.0.1",
+    "eslint": "^5.16.0",
     "finis": "^0.4.3",
+    "flash-store": "^0.16.6",
+    "grpc": "^1.24.2",
     "read-pkg-up": "^7.0.0",
-    "wechaty-puppet-padplus": "^0.1.0",
-    "wechaty": "^0.29.8"
+    "rocksdb": "^4.1.0",
+    "wechaty": "^0.29.8",
+    "wechaty-puppet": "^0.15.28",
+    "wechaty-puppet-padplus": "^0.1.3"
```

Comment:
It's also not the best practice to dependence the `wechaty-puppet-padplus` module: it would be better to be managed by `wechaty` itself.